blockware/fireworks/lib/Particle/Particle.cpp

```diff
@@ -17,7 +17,7 @@ Particle::Particle(int x, int y, bool isRocket, int color)
     _vel.set(random(-3, 3), random(-7, -3));
     _resistance = 1.0f;
   } else {
-    float angle = random(TAU);
```

Comment:
For the future there's also these `random` helpers but they don't really do much: 

https://github.com/bountylabs/blocks-with-screens/blob/main/blockware/lib/Random/Random.h